Source/PeleLM.cpp

```diff
@@ -6604,7 +6604,7 @@ PeleLM::compute_scalar_advection_fluxes_and_divergence (const MultiFab& Force,
 
   MultiFab edgeflux[AMREX_SPACEDIM];
   MultiFab edgestate[AMREX_SPACEDIM];
-  int nghost = 2;
+  int nghost = 4;
 
```

Comment:
Having numbers harder like this scares me.  This is 2 original + 2 for EB?  Or what.  is there a way to lift this out a bit and make it more obvious what's happening?

Reply:
fixed

Reply:
Same comment as above...it would be good to document why the correct number is 4 here.

Reply:
Explain where the 4 comes from for here as well.